es6/timeline.js

```diff
@@ -69,6 +121,28 @@ class TimelineComponent extends React.Component {
 
     render() {
         const {queue, warped_time, debug} = this.props
+        const anim_list = []
+        let container_width = 0
+        for (let idx in queue){
+            anim_list.push(AnimRow(
```

Comment:
This should just be a map, not a for-loop with array.push:
```javascript
anim_list.push(<AnimRow anim={anim} idx={idx} scale={this.state.scale} warped_time={warped_time}/>)
```

Reply:
@pirate this is a for loop because it has some side effects like calculate the container_width. That was one of you recomendations

Reply:
Oh whoops, I edited this comment minutes after making it, but I must've forgotten to hit save.